R/sidebar.R

```diff
@@ -127,6 +133,13 @@ layout_sidebar <- function(
     columns_collapse <- rev(columns_collapse)
   }
 
+  grid_template_rows_mobile <-
+    if (is.null(sidebar$max_height_mobile)) {
+      if (is.null(height)) "fit-content(250px)" else "fit-content(50%)"
+    } else {
+      sprintf("fit-content(%s)", sidebar$max_height_mobile)
```

Comment:
Just pass the value and use `fit-content(var(...))`  in the css

Reply:
e93be1306fec45bba98007d750d580ac28ba950c